CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## Release 0.8.0
+
+New features:
+- Show message in the UI when topologies exceed size limits.
+  [#474](https://github.com/weaveworks/scope/issues/474)
+- Provide container image information in detail pane for containers.
+  [#398](https://github.com/weaveworks/scope/issues/398)
+- When filtering out system containers, also filter out pseudo nodes, if they were only connected to system containers.
+  [#483](https://github.com/weaveworks/scope/issues/483)
+- Show number of filtered nodes in status pane.
+  [#509](https://github.com/weaveworks/scope/issues/509)
+- Improve unit and integration test coverage.
```

Comment:
We haven't typically mentioned testing improvements in changelogs, just user visible changes.